### DISCUSSION TITLE
Is it possible to subscribe the click event on a specific `problem` item?

### DISCUSSION THREAD

**Discussion by xujuntwt95329** [microsoft/vscode-discussions#1035] [Extension Development QnA] [answered]
When clicking the items inside problem panel, VSCode will directly open the corresponding file. This won't work with some graphical editors. So I‘m wondering is there any API or workaround that I can have my customized handler for the problem item click event?

I have tried the `code action provider` as suggested in https://stackoverflow.com/questions/72216574/vscode-problems-tab-how-to-subscribe-to-onclick-event-on-one-specific-problem, but the experience may be not friendly enough for end users.

**Comment by isidorn** [comment, marked as answer]
No, there is no API to have a customized handler for the problem item click event.

**Reply by xujuntwt95329** [reply]
Thanks for the reply!

Is there any workaround to finish such feature? It seems graphical editor is widely used in many scenarios.

**Reply by isidorn** [reply]
No workarounds. VS Code API is not designed in such a way.
